powerVCF.psm1

```diff
@@ -1877,16 +1874,301 @@ Function Set-VCFCeip {
+		    $response.elements
+        }
+    catch {
+        # Call the function ResponseExeception which handles execption messages
+        ResponseExeception
+    }
+}
+Export-ModuleMember -Function Get-VCFCertificateAuthConfiguration
+
+######### End Certificate Configuration Operations ##########
+
+######### Start Certificate Configuration Operations ##########
+
+Function Get-VCFDepotCredentials {
+<#
+    .SYNOPSIS
+    Get Depot Settings
+	
+    .DESCRIPTION
+     Retrieves the configuration for the depot of the connected SDDC Manager
+	
+    .EXAMPLE
+    This example shows credentials that have been configured for the depot.
+	
+	PS C:\> Get-VCFDepotCredentials	
+#>
+
+    $headers = @{"Accept" = "application/json"}
+    $headers.Add("Authorization", "Basic $base64AuthInfo")
+    $uri = "https://$sddcManager/v1/system/settings/depot"
+
+    try { 	
+		    $response = Invoke-RestMethod -Method GET -URI $uri -headers $headers
+		    $response
+        }
+    catch {
+        # Call the function ResponseExeception which handles execption messages
+        ResponseExeception
+    }
+}
+Export-ModuleMember -Function Get-VCFDepotCredentials
+
+Function Set-VCFDepotCredentials {
+<#
+    .SYNOPSIS
+    Update the Depot Settings
+	
+    .DESCRIPTION
+     Update the configuration for the depot of the connected SDDC Manager
+	
+    .EXAMPLE
+    This example sets the credentials that have been configured for the depot.
+	
+	PS C:\> Set-VCFDepotCredentials -username "user@yourdomain.com" -password "VMware1!"
+#>
+
+	Param (
+		[Parameter (Mandatory=$true)]
+        [string]$username,
+		[Parameter (Mandatory=$true)]
+        [string]$password
+    )
+
+    $headers = @{"Accept" = "application/json"}
+    $headers.Add("Authorization", "Basic $base64AuthInfo")
+    $uri = "https://$sddcManager/v1/system/settings/depot"
+    try {
+
+        if ( -not $PsBoundParameters.ContainsKey("username") -and ( -not $PsBoundParameters.ContainsKey("password"))){
+			throw "You must enter a username and password"
+		}
+        $ConfigJson = '{"vmwareAccount": {"username": "'+$username+'","password": "'+$password+'"}}'
+        $response = Invoke-RestMethod -Method PUT -URI $uri -ContentType application/json -headers $headers -body $ConfigJson
+        $response      
+    }
+    catch {
+        # Call the function ResponseExeception which handles execption messages
+        ResponseExeception
+    }
+
+}
+Export-ModuleMember -Function Set-VCFDepotCredentials
+
+######### End Certificate Configuration Operations ##########
+
+######### Start SDDC Manager Operations ##########
+
+Function Get-VCFManager {
+<#
+    .SYNOPSIS
+    Get the Sddc Managers
+	
+    .DESCRIPTION
+     Retrieves the detials for SDDC Manager
+	
+    .EXAMPLE
+    This example shows how to get the list of SDDC Managers.
+	
+	PS C:\> Get-VCFManager
+
+    This example shows how to return the details for a specic SDDC Manager based on the ID
```

Comment:
If you dont pass -id will it retrieve more than 1 if federated?

Reply:
Unknown as I don't have federation environment at this point